### DISCUSSION TITLE
Add `install-cni` command to binary?

### DISCUSSION THREAD

**Discussion by akhilles** [k8snetworkplumbingwg/whereabouts#235] [Ideas]
Any thoughts on adding an `install-cni` command to the binary that basically absorbs `./script/install-cni.sh`? An example of this in practice: https://github.com/istio/istio/blob/master/cni/pkg/cmd/root.go#L48.

This would allow the container image to be distroless by removing the dependencies on coreutils/busybox.

**Comment by maiqueb** [comment]
I don't think the binary should care about installation.

I would be in favor of deprecating that script though - making the installation based on on an init container. 

Of course ... this would only be a possibility if we only cared about kubernetes. 

... not sure how that plays along with our claim of:
> Whereabouts is designed with Kubernetes in mind, but, isn't limited to use in just Kubernetes.
